src/aqua_diagnostics/timeseries/base.py

```diff
@@ -420,10 +427,10 @@ def save_plot(self, fig, var: str = None, description: str = None, region: str =
         metadata = {"Description": description, "dpi": dpi }
         extra_keys = {'diagnostic_product': diagnostic_product}
 
-        if var is not None:
-            extra_keys.update({'var': var})
-        if region is not None:
-            region = region.replace(' ', '').lower()
+        if self.standard_name is not None:
+            extra_keys.update({'var': self.standard_name})
```

Comment:
This creates filenames including standard_name instead of var. Actually I believe this should be discussed.... I sort of like and think it is pretty compact dealing filenames just with the short name, no?

Reply:
If self.standard name is None (I do not see how, since you fix it earlier) no var info is include din the filename?

Reply:
I did not yet give approval because I believe we should settle this